app/models/manageiq/providers/amazon/network_manager/refresh_parser.rb

```diff
@@ -323,12 +323,6 @@ def parse_load_balancer_health_check(lb)
     port         = target_match[2].to_i
     url_path     = target_match[3]
 
-    matched_listener = @data.fetch_path(:load_balancer_listeners).detect do |listener|
```

Comment:
ok, but why don't we need it?
now it looks like we dont save `loadbalancer_listeners` anymore